tutorial/tutorial-template-guide.md

```diff
@@ -0,0 +1,132 @@
+
+Tutorials are often confused with how-to guides because they help users achieve a specific goal. However, how-to guides assume that your users have some experience with your product or are familiar with certain concepts. 
+
+In comparison, tutorials give your users hands-on experience with your product so that they can understand key concepts about your product in a more practical context. Hands-on learning often helps learners gain a deeper understanding of the product by helping them see how they can use a tool to accomplish a task they care about.
+
+## Helpful tips when writing your tutorial 
+
+* Keep your tutorial focused. If needed, include links to other pieces of documentation that explain concepts or terms in more depth.
+* Lengthy tutorials can overwhelm users. Consider condensing or removing steps or reevaluating the scope of the tutorial.
+* If users will copy and paste code samples, make sure they include the following:
+  * Any required `import` or `using` statements
+  * Code comments that explain what the code does 
+
+## About the tutorial template 
+
+This tutorial can be used in one of two ways: 
+* For providing a walkthrough of a particular feature.
+* Walking users through completing a small project. 
+
+Ideally, your users already completed a quickstart or "getting started" tutorial and have a general understanding of your product.  
+
+### About the "Overview" section
+
+The overview section is important, as it can motivate your users to begin their learning journey with your product and help set them up for success. There are three topics you should cover in this section: learning objectives, intended audience, and any prerequisite background knowledge. 
+
+One of the first things users want to know when they begin a tutorial is, "What will I be able to do? What knowledge will I gain by completing this tutorial?" Well-written learning objectives can help to answer these questions. **Learning objectives** describe the specific skills or knowledge users will be able to demonstrate after completing your tutorial.
+
+Before you start writing your tutorial, develop some learning objectives. After you identify *what* skills and knowledge you want users to gain from your tutorial, you can start planning out the content itself, using the learning objectives to help you decide what topics are in scope, and which ones are out of scope. 
+
+Learning objectives usually start off with, "By the end of this tutorial, you will be able to...", are followed by a phrase that starts with a verb like *design*, *assess*, or *develop*.
+
+Here are some examples:
+
+* By the end of this tutorial, you'll be able to define components in React.
+* By the end of this tutorial, you'll be able to perform common tasks in Google Drive, such as viewing, editing, and saving documents.
+* By the end of this tutorial, you'll be able to create a private channel in Slack.
+
```

Comment:
**Suggestion (non-blocking):** You could also mention they could have more than one objective. If they do, they should consider formatting it in a bulleted list.

**Chore (style question):** We should also decide whether we represent bullets using asterisks (*) or dashes (-) in our tutorials so that we can be consistent. I personally like dashes.